src/main/java/uk/gov/ons/response/survey/representation/SurveyDTO.java

```diff
@@ -15,4 +15,5 @@
   private String shortName;
   private String longName;
   private String surveyRef;
+  private String legalBasis;
```

Comment:
Does this need a test?

Reply:
All the getters and setters for this class are done through lombock, so i'm guessing no. Also the tests probably make more sense in the services which consume the JSON response.